tests/e2e/csi_cns_telemetry.go

```diff
@@ -191,6 +191,12 @@ var _ = ginkgo.Describe("[csi-block-vanilla] [csi-file-vanilla] [csi-block-vanil
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		}()
 
+		// For Old PVCs to reflect the latest Cluster-Distribution Value, wait for
```

Comment:
instead of static sleep let's add a call to force trigger a full sync via the crd